movai_core_shared/envvars.py

```diff
@@ -48,7 +48,8 @@
 ROS2_DISTRO = "dashing"
 # ZMQ environment variables
 MSG_HANDLER_LOCAL_CONN = os.getenv("MSG_HANDLER_LOCAL_CONN", "ipc:///run/movai/msg_handler_local_comm.ipc")
-MOVAI_ZMQ_TIMEOUT_MS = os.getenv("MOVAI_ZMQ_TIMEOUT_MS", 1000)
+MOVAI_ZMQ_RECV_TIMEOUT_MS = os.getenv("MOVAI_ZMQ_RECV_TIMEOUT_MS", 5000)
```

Comment:
maybe 5 seconds is too long 
the HTML request from the F.E will be set to NO RESPONSE by 5 seconds

Reply:
MS here is misleading between milliseconds and microseconds